Packages/manifest.json

```diff
@@ -48,5 +52,19 @@
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.wind": "1.0.0",
     "com.unity.modules.xr": "1.0.0"
-  }
+  },
+  "scopedRegistries": [
```

Comment:
should this still be here now that UTP is merged?

Reply:
fixed